app/src/main/java/edu/cascadia/mobile/apps/movies/MainActivity.java

```diff
@@ -21,24 +22,31 @@
 import edu.cascadia.mobile.apps.movies.model.MovieEntity;
 import edu.cascadia.mobile.apps.movies.ui.MoviesAdapter;
 import edu.cascadia.mobile.apps.movies.utilities.SampleData;
+import edu.cascadia.mobile.apps.movies.viewmodel.MainViewModel;
 
 public class MainActivity extends AppCompatActivity {
 
     RecyclerView mRecyclerView;
     MoviesAdapter mMoviesAdapter;
-    private movieDatabase mDatabase;
+    public movieDatabase mDatabase;
+    //public MainViewModel mViewModel;
+    private List<MovieEntity> movieData = new ArrayList<>();
 
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_main);
+        final MainViewModel mViewModel = ViewModelProviders.of(this).get(MainViewModel.class);
         Toolbar toolbar = findViewById(R.id.toolbar);
         setSupportActionBar(toolbar);
         mRecyclerView = findViewById(R.id.recycler_view);
 
+
+
         //Get Database
         mDatabase = movieDatabase.getInstance(this);
-        mDatabase.movieDao().addAll(SampleData.getMovies());
+        movieData.addAll(mViewModel.mMovies  );
+
 
         initRecyclerView();
 
```

Comment:
Following the sample, you would add an initViewModel() call here and uncomment your definition below